src/main/java/org/tailormap/api/scheduling/TaskManagerService.java

```diff
@@ -39,6 +41,39 @@ public TaskManagerService(@Autowired Scheduler scheduler) {
     this.scheduler = scheduler;
   }
 
+  /**
+   * Create a one-time job and schedule it to start immediately.
+   *
+   * @param job the task class to create
+   * @param jobData a map with job data, the {@code type} and {@code description} keys are mandatory
+   * @return the task name, a hash of the description
+   * @throws SchedulerException if the job could not be scheduled
+   */
+  public String createTask(Class<? extends QuartzJobBean> job, TMJobDataMap jobData)
+      throws SchedulerException {
+    final String taskName = DigestUtils.md5Hex(jobData.getDescription());
```

Comment:
Why not just use a new UUID? Then we don't have to think about the description having to be unique to avoid using the same `taskName`.

Reply:
that will allow a user to keep on adding jobs for any index that does not have a schedule... in hindsight this will also prevent stopping/deleting the task in the task admin